package.json

```diff
@@ -18,10 +18,10 @@
     "string"
   ],
   "author": "Hanson Wang (hanson.wng@gmail.com)",
-  "repository": "https://github.com/atom/fuzzy-native",
+  "repository": "https://github.com/pulsar-edit/fuzzy-native",
   "license": "MIT",
   "dependencies": {
-    "nan": "^2.14.2"
+    "nan": "2.17.0"
```

Comment:
Can we use keep using caret semver here?

---

Pinned `nan` versions can be a pain, requiring new versions of packages to be published to get the newer `nan` version. I usually really like when `nan` is specified with pretty loose semver constraint such as caret `^`.

It would also be possible to revert this line and ensure on the "consumer" side of this package that `nan` is always 2.17.0 or newer. Since the previous caret semver range `^2.14.2` would have allowed/included 2.17.0 as valid.

But bumping the minimum kinda makes sense if it needs it on newer Electron.

Reply:
Wow, I didn't realize these commits are all the way back from May, 6 months ago.

So, maybe not so urgent to address this, but caret `^` semver is preferable IMO, if you get a chance. Or I can do it myself pretty easily, just a one-character diff.

And then we can bump to the updated commit in Pulsar core repo.